packages/react/src/button/Button.tsx

```diff
@@ -15,38 +15,70 @@ import { Stack } from "../stack";
 import { Text } from "../text";
 import { type Recipe, recipe } from "./Button.recipe";
 
+const presets = {
+  basic: { colorScheme: "secondary", variant: "solid" },
+  basicLink: { colorScheme: "secondary", variant: "link" },
+  danger: { colorScheme: "danger", variant: "solid" },
+  dangerOutline: { colorScheme: "danger", variant: "outline" },
```

Comment:
let's do dash case here as well
```suggestion
  "danger-outline": { colorScheme: "danger", variant: "outline" },
```